app/controlplane/pkg/data/ent/migrate/migrations/20250908160222.sql

```diff
@@ -0,0 +1,4 @@
+-- Drop index "workflowrun_state_created_at" from table: "workflow_runs"
+DROP INDEX "workflowrun_state_created_at";
```

Comment:
change it to concurrently, see the annotation and the CONCURRENTLY stanza 
for example
```sql
-- atlas:txmode none

DROP INDEX workflowrun_attestation_digest;
CREATE INDEX CONCURRENTLY "workflowrun_attestation_digest" ON "workflow_runs" ("attestation_digest");
```